docs/index.html

```diff
@@ -7,56 +7,49 @@
 <div class="panel panel-info">
     <div class="panel-heading">
         {% include icon.html type="info" %}
-        ev3dev is a community-maintained project that relies on people's contributions.
+        ev3dev is a community-maintained project that relies on community contributions.
```

Comment:
Ahhh good idea! I spent more time that I am willing to admit attempting to find a better way to phrase this.